velox/docs/functions/presto/conversion.rst

```diff
@@ -289,6 +304,24 @@ supported conversions to/from JSON are listed in :doc:`json`.
      -
      - 
      - 
+     - Y
+   * - ipprefix
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - Y
```

Comment:
If I count this right, this means conversion to VARCHAR is allowed but not VARBINARY? Wouldn't we also allow VARBINARY just like for IPADDRESS?
Or is that not supported by Presto?

Reply:
It's currently not supported in presto.